<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>785</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>831</width>
    <height>779</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>1024px-Weather-few-clouds.svg.png</normaloff>1024px-Weather-few-clouds.svg.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
      <item>
       <widget class="QLabel" name="pic_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>781</width>
          <height>521</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>999999</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Device</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dev_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="currentText">
               <string>2</string>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,4,1,4">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Date Range</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="date_start">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2019</year>
                <month>8</month>
                <day>6</day>
               </datetime>
              </property>
              <property name="time">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::MonthSection</enum>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>to</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="date_end">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>23</hour>
                <minute>59</minute>
                <second>59</second>
                <year>2019</year>
                <month>8</month>
                <day>6</day>
               </datetime>
              </property>
              <property name="time">
               <time>
                <hour>23</hour>
                <minute>59</minute>
                <second>59</second>
               </time>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::DaySection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Parameter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sel_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Temperature</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wind Speed</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Soil Moisture</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Soil Temperature</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Humidity</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pressure</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Voltage</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wind Direction</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="refreshbutton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Refresh Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="MAC_LOGO">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Save_Image"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="menuGraph_Resolution">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Graph Resolution</string>
     </property>
     <addaction name="action200_dpi"/>
     <addaction name="action300_dpi"/>
     <addaction name="action400_dpi"/>
     <addaction name="action600_dpi"/>
     <addaction name="separator"/>
     <addaction name="current_dpi"/>
    </widget>
    <addaction name="menuGraph_Resolution"/>
    <addaction name="actionAdd_Device"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
  </widget>
  <action name="Save_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action200_dpi">
   <property name="text">
    <string>200 dpi</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action300_dpi">
   <property name="text">
    <string>300 dpi</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action400_dpi">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>400 dpi</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action600_dpi">
   <property name="text">
    <string>600 dpi</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="current_dpi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Default is 200 dpi</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionAdd_Device">
   <property name="text">
    <string>Add Device</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
